### PR TITLE
adding note for migrating tooltip with popover

### PR DESCRIPTION
**Pull request by TalZa1810**
Adding a disclaimer for migrating white `<Tooltip/>` to `<Popover/>`

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -1402,6 +1402,10 @@ Tooltip no longer supports theme white, but white skin is available on Popover w
 > Warning #2: White Tooltip should be used only in cases when app background is dark - all other cases should have dark Tooltip.
 
 
+> Note:
+> - `content` prop should be replaced with `<Popover.Content/>`.
```
Comment (mykas) [inline MIGRATION.md]: Thats a bit strange: Why `content` prop should be replaced with `<Popover.Content/>` ? Maybe would be more understandable `content` prop value should be used inside `<Popover.Content/>` or I did not get something here?

Comment (TalZa1810) [reply MIGRATION.md]: yeah, that's what I meant :) I'll re-write this :)

Comment (TalZa1810) [reply MIGRATION.md]: > Note:
> - `content` prop value should be used within `<Popover.Content/>`.
> - `children` prop value should be used within `<Popover.Element/>`.

Is it a better phrasing in your opinion? @mykas

Comment (mykas) [reply MIGRATION.md]: yep :)

File: MIGRATION.md [hunk 2]
```diff
@@ -1402,6 +1402,10 @@ Tooltip no longer supports theme white, but white skin is available on Popover w
 > Warning #2: White Tooltip should be used only in cases when app background is dark - all other cases should have dark Tooltip.
 
 
+> Note:
```
Comment (shlomitc) [inline MIGRATION.md]: @TalZa1810 @mykas - maybe just add the "Before" example like in the other cases?

`<Tooltip skin="light"/>

**Pull request merged**